app/views/performance_dashboards/overview/breakdown/_base.haml

```diff
@@ -22,23 +22,58 @@
     }
   }
 
-.d-flex.flex-wrap.justify-content-between.align-items-end.mb-2
-  %header.mb-2.mr-4{style: 'flex: 1; min-width: 300px'}
+.d-flex.flex-wrap.justify-content-between.mb-2{class: @report_variant == 'sparse' ? 'align-items-baseline' : 'align-items-end'}
+  %header.mb-2.ml-auto{style: 'flex: 1; min-width: 300px'}
     %h2
-      = [key.to_s.titleize, @breakdown_section_title].compact.join(': ')
+      - if @breakdown_section_title
+        = [key.to_s.titleize, @breakdown_section_title].compact.join(': ')
+      - elsif key == :enrolled && @report_variant == 'sparse'
+        Actively Enrolled Clients from #{@report.date_range_words}
+      - else
+        = key.to_s.titleize
```

Comment:
@ttoomey, this doesn't appear to be working correctly in the PDF version.  Can you take a look?

Reply:
ah yes, I wasn't sure if these changes should be carried into the PDF so I left it alone. Should be simple to fix, just need to set the variant ivar.